solr/core/src/test/org/apache/solr/security/BasicAuthIntegrationTest.java

```diff
@@ -232,6 +229,17 @@ public void testBasicAuth() throws Exception {
         log.error("RunExampleTool failed due to: " + e +
             "; stdout from tool prior to failure: " + baos.toString(StandardCharsets.UTF_8.name()));
       }
+      
+      addDocument("harry","HarryIsUberCool","id", "5");
+
+      // Validate forwardCredentials
+      assertEquals(1, executeQuery(params("q", "id:5"), "harry", "HarryIsUberCool").getResults().getNumFound());
+      // TODO: Validate that this requests were handled by PKI plugin
+      executeCommand(baseUrl + authcPrefix, cl, "{set-property : { forwardCredentials: true}}", "harry", "HarryIsUberCool");
+      verifySecurityStatus(cl, baseUrl + authcPrefix, "authentication/forwardCredentials", "true", 20, "harry", "HarryIsUberCool");
+      assertEquals(1, executeQuery(params("q", "id:5"), "harry", "HarryIsUberCool").getResults().getNumFound());
+      // NOCOMMIT: Validate that sub requests were handled by BasicAuth plugin
```

Comment:
After committing SOLR-12791 we can test this by asserting that metrics has been incremented for Basic and not for PKI

Reply:
Done